hilti/toolchain/src/compiler/driver.cc

```diff
@@ -73,7 +73,7 @@ Driver::Driver(std::string name, const hilti::rt::filesystem::path& argv0) : _na
 Driver::~Driver() {
     if ( _driver_options.report_times )
         try {
-            util::timing::summary(std::cerr);
+            util::timing::summary(std::cout);
```

Comment:
This one I'd prefer to stay on stderr, the reason being that it's augmenting any normal output to stdout, so that the two don't get mixed together.